docs/fastdds/xml_configuration/domainparticipant.rst

```diff
@@ -672,12 +662,21 @@ Send buffers
      - ``uint32_t``
      - 0
    * - ``<dynamic>``
-     - Whether the number of send buffers is allowed to grow. |br|
-       See :ref:`sendbuffersallocationattributes`.
+     - Whether the number of send buffers is allowed to |br|
+       grow. See :ref:`sendbuffersallocationattributes`.
      - ``bool``
      - false
+   * - ``<network_buffers_config>``
+     - Network buffer :ref:`CommonAlloc` to specify the |br|
+       number of network buffers to be allocated for each |br|
+       send buffer. See :ref:`ResourceLimitedContainerConfig`.
+     - :ref:`CommonAlloc`
+     - (16, inf, 16)
 
 .. note::
     The default value ``0`` of ``<preallocated_number>`` will perform an initial guess of the number of buffers
     required, based on the number of threads from which a send operation could be started.
     So it does not mean there are no buffers, instead it would use the maximum amount of buffers available.
+    On the contrary, ``<network_buffers_config>`` will default to an initial number of 16 buffers, with an infinite
+    maximum and an increment of 16 buffers per send buffer. Setting a value of ``0`` will prevent the creation of
+    network buffers. In case of doubt, leave it unset.
```

Comment:
```suggestion
    maximum and an increment of 16 buffers per send buffer.
    A value of ``0`` will imply more dynamic allocations, especially at the beginning of the execution.
    In case of doubt, it should be left to the default values.
```